.github/workflows/translation_statistics.yml

```
@@ -23,7 +23,7 @@ jobs:
     env:
       TRANSIFEX_PASSWORD: ${{ secrets.TRANSIFEX_PASSWORD }}
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      BACKPORT_BRANCH: "release_3.28"
+      LABELS: "Translation, backport release_3.28, backport release_3.34"
```

```suggestion
      LABELS: "translation, backport release_3.28, backport release_3.34"
```

Or was that capitalized on purpose?

---

To match the existing label (unless gh is case-insensitive)